src/web/data/userMapData.json

```diff
@@ -1900,19 +2217,23 @@
       "Texas A&M University",
       "Texas A&M University-Corpus Christi",
       "Texas Lutheran University",
+      "Texas State University, San Marcos",
       "Texas Southern University",
       "Texas Tech University",
       "Texas Woman's University",
       "Thales University",
       "The Bromfield School",
       "The Bush School",
+      "The Chicago School of Professional Psychology",
+      "The College of New Jersey",
       "The College of Mount Saint Vincent",
       "The College of Wooster",
       "The George Washington University",
       "The Hun School of Princeton",
       "The Lake Erie College of Osteopathic Medicine (LECOM)",
       "The Master's School",
       "The New School",
+      "The Ohio State University",
```

Comment:
Duplicate entry. We already have Ohio State University.

Reply:
Removed